requirements.txt

```diff
@@ -11,8 +11,8 @@ git+https://github.com/dwf/picklable_itertools.git#egg=picklable-itertools
 
 --allow-external theano
 --allow-unverified theano
-git+https://github.com/Theano/Theano.git#egg=theano
+git+https://github.com/Theano/Theano.git@c6d2ab3929182cdf7cf12e4b1f191a6ff99d07cc#egg=theano
 
 --allow-external fuel
 --allow-unverified fuel
-git+https://github.com/mila-udem/fuel#egg=fuel
+git+https://github.com/mila-udem/fuel@stable#egg=fuel
```

Comment:
This is the right way to go, but if we want to keep outdated versions installable (and I think we want), we will have to create revisions pointing to a tag or exact commit in Fuel.  This is in a way the minimal form of maintenance of outdated versions. We can not afford more than that, I guess, with our limited manpower.